### DISCUSSION TITLE
Kind of 180 deg reverse directed offtopic topic...kind of: Terminal setup

### DISCUSSION THREAD

**Discussion by mcctuxic** [tabemann/zeptoforth#129] [General]
Hi Travis,

sorry, this is kind of backward on topic or simply: Offtopic.

Until now I used  a GENTOO Linux with Openbox with Kitty with picocom (wait for it...!) and finally (and most important:
*Z*E*P*T*O*F*O*R*T*H* 8)

But from time to time things became weird and I need a short look into screen and a reset-command and sometimes a reboot
of zeptoforth and new instance of kitty.

Then I tried alacritty with tmux and until now it seems to behave better in a way I want forget kitty and use alacritty instead.

But there is one thing I can't do currently:
Wenn Openbox starts, it automagically starts (manually configured) neomutt (mailer based on ncurses) within - previously -
kitty. Kitty has internal support vor v/h splits, tabs and so on, which needs to be done by tmux now.

So....alacritty needs to start tmux, which needs to start neomutt..

Logical result

alacritty -e tmux -c neomutt

this command has only one disadvantage: It does not work ;)

It looks like tmux is not running and alacritty runs neomutt directly.

On the net are basicly 1000th of tmux/alacritty related articles about thousands of themes, colors and decorations...but couldn't find a single one, which solves my problem...

Do you have experience with this "double launcher" ?

Cheers!
Tuxic

**Comment by tabemann** [comment]
On Tue, Aug 6, 2024 at 12:34 PM mcctuxic ***@***.***> wrote:

> Hi Travis,
>
> sorry, this is kind of backward on topic or simply: Offtopic.
>
> Until now I used a GENTOO Linux with Openbox with Kitty with picocom (wait
> for it...!) and finally (and most important:
> *Z*E*P*T*O*F*O*R*T*H* 8)
>
> But from time to time things became weird and I need a short look into
> screen and a reset-command and sometimes a reboot
> of zeptoforth and new instance of kitty.
>
> Then I tried alacritty with tmux and until now it seems to behave better
> in a way I want forget kitty and use alacritty instead.
>
> But there is one thing I can't do currently:
> Wenn Openbox starts, it automagically starts (manually configured) neomutt
> (mailer based on ncurses) within - previously -
> kitty. Kitty has internal support vor v/h splits, tabs and so on, which
> needs to be done by tmux now.
>
> So....alacritty needs to start tmux, which needs to start neomutt..
>
> Logical result
>
> alacritty -e tmux -c neomutt
>
> this command has only one disadvantage: It does not work ;)
>
> It looks like tmux is not running and alacritty runs neomutt directly.
>
> On the net are basicly 1000th of tmux/alacritty related articles about
> thousands of themes, colors and decorations...but couldn't find a single
> one, which solves my problem...
>
> Do you have experience with this "double launcher" ?
>
> Cheers!
> Tuxic
>
I have never used tmux or alacritty (or for that matter kitty) myself so I
cannot be of much use. My work environment with zeptoforth is basically
using picocom under xfce4-terminal or zeptocom.js under Chrome as a
terminal (zeptoed is not compatible with zeptocom.js, hence why I have
largely been using picocom as of late) or utils/codeload3.sh or zeptocom.js
for mass uploads of code.

Travis
